net/openappid/Makefile

```diff
@@ -0,0 +1,52 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openappid
+PKG_VERSION:=26425
+PKG_RELEASE:=1
+
+PKG_SOURCE:=snort-$(PKG_NAME).tar.gz
```

Comment:
```suggestion
PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
```

Reply:
Well, the official name of the software is in fact `openappid` not `snort-openappid` so I could set `PKG_NAME:=snort-openappid` but that isn't the official upstream name.

Reply:
Well I don't see why you have to use `snort-openappid` here.  See my `PKG_SOURCE_URL` change above, it should work.

Reply:
Change made